files/en-us/web/css/calc-constant/index.md

```diff
@@ -14,55 +14,45 @@ browser-compat: css.types.calc-constant
 
 {{CSSRef}}
 
-The **`<calc-constant>`** [CSS](/en-US/docs/Web/CSS) [data type](/en-US/docs/Web/CSS/CSS_Types) represents well-defined constants such as `e` and `π`. Rather than require authors to manually type out several digits of these constants, a few of them are provided directly by CSS.
+The **`<calc-constant>`** [CSS](/en-US/docs/Web/CSS) [data type](/en-US/docs/Web/CSS/CSS_Types) represents common constants such as `e` and `pi`.
+These constants are supported for convenience in CSS so that authors do not have to manually write or calculate them.
+Mathematical constants can only be used inside [CSS math functions](/en-US/docs/Web/CSS/CSS_Functions#math_functions) for calculations.
+If used outside of a calculation, they're treated like any other keyword.
+For example:
+
+- `animation-name: pi;` refers to an animation named "pi", not the `pi` numeric constant.
```

Comment:
I'm happy with this, ish, although I like the practice of having a very short summary at the start:

> The **`<calc-constant>`** [CSS](/en-US/docs/Web/CSS) [data type](/en-US/docs/Web/CSS/CSS_Types) represents common constants such as `e` and `pi`.

...and then putting the other stuff in an H2 Description that goes after H2 Syntax.

Reply:
That makes sense, I'm happy with either

Reply:
> I like the practice of having a very short summary at the start

Me too.